src/cpp/core/include/core/Result.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Result.hpp
+ *
+ * Copyright (C) 2023 by Posit, PBC
+ *
+ * Unless you have received this program directly from Posit pursuant
+ * to the terms of a commercial license agreement with Posit, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+#ifndef CORE_RESULT_HPP
+#define CORE_RESULT_HPP
+
+#include <expected/expected.hpp>
+#include <shared_core/Error.hpp>
+
+namespace rstudio::core {
```

Comment:
```suggestion
namespace rstudio {
namespace core {
```

This syntax is a C++17 feature :(